rpm/create-rpm.sh

```diff
@@ -19,19 +19,65 @@
 # For those usages not covered by this license please contact with
 # iot_support at tid dot es
 
-PROXY_VERSION=$2
-if [ -z "$PROXY_VERSION" ]; then
-  PROXY_VERSION=0.4.0_next
+
+function usage() {
+    SCRIPT=$(basename $0)
+
+    printf "\n" >&2
+    printf "usage: ${SCRIPT} [options] \n" >&2
+    printf "\n" >&2
+    printf "Options:\n" >&2
+    printf "\n" >&2
+    printf "    -h                    show usage\n" >&2
+    printf "    -v VERSION            Mandatory parameter. Version for rpm product preferably in format x.y.z \n" >&2
+    printf "    -r RELEASE            Mandatory parameter. Release for product. I.E. 0.ge58dffa \n" >&2
+    printf "\n" >&2
+    exit 1
+}
+
+while getopts ":v:r:u:a:h" opt
+
+do
+    case $opt in
+        v)
+            VERSION_ARG=${OPTARG}
+            ;;
+        r)
+            RELEASE_ARG=${OPTARG}
+            ;;
+        h)
+            usage
```

Comment:
Reading again, I will remove the `exit 1` form `usage` funcion and I will add an `exit 0` in the case option. Is more clear